src/components/options-list-wrapper/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ComponentProps, ReactElement, useRef } from 'react';
+import { Animated, StyleProp, View, ViewStyle } from 'react-native';
+
+import { ANIMATION_DURATION } from '../../constants/styles';
+import type { OptionalToRequired } from '../../helpers';
+import type { Select } from '../../index';
+import type { OptionsList } from '../options-list';
+
+type FromSelectProps = Pick<ComponentProps<typeof Select>, 'isAnimated'>;
+type FromOptionListProps = Pick<ComponentProps<typeof OptionsList>, 'isOpened'>;
+
+type WrapperStyles = {
+    wrapperStyles: StyleProp<ViewStyle>;
+};
+
+type OptionsListWrapperProps = OptionalToRequired<FromSelectProps & FromOptionListProps> & {
+    children: ReactElement;
```

Comment:
I think you use here `ReactNode` instead `ReactElement` because `ReactNode` is more generic